scripts/quests/otherAreas/The_Call_of_the_Sea.lua

```diff
@@ -0,0 +1,160 @@
+-----------------------------------
+-- The Call of the Sea
+-----------------------------------
+-- Log ID: 4, Quest ID: 67
+-- Equette       !pos 0 -22 -17
+-- Leporaitceau  !pos 74 -24 5
+-- Anteurephiaux !pos 74 -24 5
+-- ???           !pos
+-----------------------------------
+require('scripts/globals/items')
+require('scripts/globals/moghouse')
+require('scripts/globals/quests')
+require('scripts/globals/status')
+require('scripts/globals/zone')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+local ID = require("scripts/zones/Misareaux_Coast/IDs")
+-----------------------------------
+local quest = Quest:new(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.THE_CALL_OF_THE_SEA)
+
+quest.reward =
+{
+    item = xi.items.MEMENTO_MUFFLER,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE
+        end,
+
+        [xi.zone.TAVNAZIAN_SAFEHOLD] =
+        {
+            ['Equette'] =
+            {
+                onTrigger = function(player, npc)
```

Comment:
Notes for this whole if quest avail section.
We should change it to use bitwise and that way we can keep it all under 1 var
quest:isVarBitsSet(player, 'Option', bitValue) and quest:setVarBit(player, 'Option', bitValue)

Examples can be seen here
https://github.com/LandSandBoat/server/blob/7e380488bc9413dc6efef568e3c01e932fb75e2f/scripts/quests/ahtUrhgan/Arts_and_Crafts.lua#L133

Then on accepted we can start the Prog var at 0

Reply:
This is amazing. Didn't know quests had bitwise. Thank you!